### PR TITLE
Fix type in web extensions + export manifest_version

### LINKED ISSUE #239: Check manifest_version equates to type
In the `manifest.json` `manifest_version` is currently only allowed to be 2. This looks like it maps to 2 as the type in `install.rdf`. We need to confirm this is the case.

### PR DESCRIPTION
**Pull request by muffinresearch**
Fix #239

### CODE REVIEW

File: src/messages/manifestjson.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import { gettext as _ } from 'utils';
+
+export const MANIFEST_VERSION_INVALID = {
+  code: 'MANIFEST_VERSION_INVALID',
+  legacyCode: null,
+  message: _('"manifest_version" in the manifest.json is not a valid value'),
+  description: _('See https://goo.gl/WyioRC (MDN Docs) for more information.'),
```
Comment (tofumatt) [inline src/messages/manifestjson.js]: Can we use a Mozilla short URL here? :wink:

**Comment by tofumatt** [comment]
r+

File: src/messages/manifestjson.js (hunk 1 above)
Comment (muffinresearch) [inline src/messages/manifestjson.js]: Yeah, do you have a link?

Comment (tofumatt) [inline src/messages/manifestjson.js]: I don’t, and as discussed in IRC it didn’t seem easy to find. I say ship this, it’s cool.

**Pull request merged**